test/TestAsset/ConfigPostProcessor/FullServiceManagerConfiguration.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+return [
+    'service_manager' => [
+        'abstract_factories' => [
+            'Zend\ServiceManager\AbstractFactory\ConfigAbstractFactory',
+        ],
+        'aliases' => [
+            'Zend\Cache\Storage\StorageInterface' => 'Zend\Cache\Storage\Adapter\Redis',
+        ],
+        'delegators' => [
+            'Zend\Cache\Storage\Adapter\Redis' => [
+                'Zend\ServiceManager\Proxy\LazyServiceFactory',
+            ],
+        ],
+        'factories' => [
+            'Zend\Form\Factory' => 'Some\Vendor\Zend\Form\ZendFormFactory',
+            'MyService' => 'Zend\ServiceManager\Factory\InvokableFactory',
+        ],
+        'initializers' => [
+            // Just for testing purposes, this initializer does not exist
+            'Zend\Form\FactoryInitializer',
+        ],
+        'invokables' => [
+            'Zend\Expressive\Router\RouterInterface' => 'MyService',
+        ],
+        'lazy_services' => [
+            'class_map' => [
+                'Zend\Cache\Storage\Adapter\Redis' => 'Zend\Cache\Storage\Adapter\Redis',
+            ],
+        ],
+        'services' => [
+            // NOTE: this is an invalid configuration, you have to provide service instances in this array.
+            // For testing purposes, we have to change this to a string we can use to verify that the value
+            // is not touched at all. Using instances would lead to
+            'Zend\Cache\Storage\Adapter\Redis' => 'Zend\Cache\Storage\Adapter\Redis',
```

Comment:
As noted in the comment above the value here is wrong.
I believe non-string values should be left as-is (it should be verified).

Also: _I think_ as we are replacing key (alias?) we want to add alias (oldKey => newKey) at this point?

Reply:
Exactly, but I had to use as string to verify that its not gonna change.
Thus, in the `FullServiceManagerConfiguration.php.out`, I check if the value is still untouched (which is).

The unit test fails now as the special `services` handling is stripped away due to the only handling for `aliases`, `invokables` and `factories`.

Reply:
> Also: I think as we are replacing key (alias?) we want to add alias (oldKey => newKey) at this point?
Yeah, that would be a good one, if the alias does not exist.

Reply:
I've re-added the special `services` handling and adding an alias for a replaced service.